packages/docs/getting-started/running-a-validator-in-mainnet.md

```diff
@@ -191,21 +191,15 @@ The `us.gcr.io/celo-org/celo-node:mainnet` image contains the [genesis block](ht
 
 ### Start your Accounts node
 
-Next, we'll run a node on your local machine so that we can use these accounts to lock CELO and authorize the keys needed to run your validator. To do this, we need to run the following commands, which fetch the genesis block and a list of other nodes in the network to connect to.
-
-```bash
-# On your local machine
-docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE init /celo/genesis.json
-export BOOTNODE_ENODES=`docker run --rm --entrypoint cat $CELO_IMAGE /celo/bootnodes`
-```
+Next, we'll run a node on your local machine so that we can use these accounts to lock CELO and authorize the keys needed to run your validator. To do this, we need to run the following command to run the node.
 
 To run the node:
```

Comment:
Maybe this line sounds a bit repetitive based on the last line.

Reply:
Thanks, I removed it.